batch2/create-batch-tables.sql

```diff
@@ -33,6 +33,8 @@ CREATE TABLE IF NOT EXISTS `batches` (
   `n_failed` INT NOT NULL DEFAULT 0,
   `n_cancelled` INT NOT NULL DEFAULT 0,
   `time_created` TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+  `time_completed` TIMESTAMP,
```

Comment:
I think all timestamps should be doubles in the database in unix time.  I'm surprised time_created is still like that, I thought I changed it.